transports/httpclient/src/main/java/org/commonjava/maven/galley/transport/htcli/util/HttpUtil.java

```diff
@@ -48,6 +53,11 @@ public static void cleanupResources( final CloseableHttpClient client, final Htt
             {
                 ( (AbstractExecutionAwareRequest) request ).reset();
             }
+            if ( request instanceof HttpPut )
+            {
+                HttpEntity e = ( (HttpPut)request ).getEntity();
+                EntityUtils.consumeQuietly( e );
+            }
```

Comment:
It seems this cleanup is used in the finally clause of http requests. I am not quite sure why the cleanup is needed because the req/resp objects are ephemoral and will be GC-ed. But this looks harmless. If we do it for httpput, should we handle the httppost too?

Reply:
I agree with you. After checking the callers they are all working with ephemeral streams. I'll close this PR.